### PR TITLE
Stories/ecer 1791

### PR DESCRIPTION
**Pull request by SoLetsDev**
ECER-1791: Update validation in application wizard to always validate on save and continue

## Description

- Moved validation to occur on each step including education + work experience. 
- Note with education added in feedback for number of education based on program requirement. 
- Work experience, user must enter at least 500 hours or 6 work experiences. 
- Removed validation checks in preview cards since user should never be in an error state here. 
- Removed validation state + all stage validation from wizard.ts. 
- HasDuplicates check moved back to the character + work experience components. 
- Application.vue - removed check that information is in a valid state. User should always be in a valid state at this point.

## Random bug fix
- Added security content policy for recaptcha to load recaptcha script. 

## Checklist
- [x] I have tested these changes locally.
- [ ] Changes to backend endpoints are covered by passing integration tests.
- [ ] I have added or updated the necessary documentation.

## Screenshots (if applicable)
Duplicates check (will disappear when user changes the input)
image
Extra input check for education
image

## Additional Comments (optional)
Any additional context or information that might be helpful for the reviewer.

### CODE REVIEW

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts [hunk 1]
```diff
@@ -45,35 +43,6 @@ export const useWizardStore = defineStore("wizard", {
     currentStepStage(state): Components.Schemas.PortalStage | ReferenceStage {
       return this.steps[state.step - 1].stage;
     },
-    validationState(state): PortalStageValidation {
```
Comment (pcatkins) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts:48]: Do we need `hasDuplicateReferences` in application store still?

**Review by pcatkins — CHANGES_REQUESTED** [review]
Looks good, see question on clean up of application store. Cool use of hidden `<v-input>` to block submission of form!

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts (hunk 1 above)
Comment (SoLetsDev) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts:48]: Ahh good catch, we don't.

Comment (SoLetsDev) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts:48]: removed.

**Comment by SoLetsDev** [comment]
Yeah, learned the trick when we had to make our own inputs for radios + select all dropdowns. :D

**Review by pcatkins — APPROVED** [review]

**Pull request merged**